Sai Ganesh Manda IBM Challenge Fall 2022 (All labs completed!)

mvsg2 opened the pull request:
It's been one hell of an experience meeting (well, not in person) my future self and saving my crew members from the otherwise inevitable doom of being sucked into the black hole!
Thanks, IBM, for this wonderful challenge wtith such an invloving storyline and the cool coding problems for using quantum computers to devise this escape plan!!
Look forward to participating in many more challenges like this which put all my limits of understanding to the test...

mvsg2 committed:
Add files via upload

Lab1 of the IBM Quantum Challenge Fall 2022

PS. The hidden encoded message is "Captian's Log"...

mvsg2 committed:
Add files via upload

Lab2 of the IBM Quantum Challenge Fall 2022

mvsg2 committed:
Delete lab1_Completed.ipynb

mvsg2 committed:
Delete lab2_Completed.ipynb

mvsg2 committed:
Add files via upload

Lab1 of the IBM Quantum Challenge Fall 2022.

PS.  The hidden encoded message is "Captian's Log"...

mvsg2 committed:
Add files via upload

Lab2 of the IBM Quantum Challenge Fall 2022

mvsg2 committed:
Add files via upload

So glad to have finally escaped the gravitaional grasp of the ravenous black hole! 
Really thankful for the quantum computers for thier unprecedented help in making this feat happen!

mvsg2 committed:
Add files via upload

Finally used some knowledge on Graph Theory Algorithms to make a successful slingshot around the planet!

mvsg2 committed:
Merge branch 'qiskit-community:main' into main

mvsg2 committed:
Merge branch 'qiskit-community:main' into main

desireevl commented:
Congratulations! Are you interested in submitting your score for the final problem in lab 4 to the unofficial leaderboard?

mvsg2 commented:
Yes, I am interested! What should I do?

On Mon, Nov 28, 2022 at 7:33 AM Desiree ***@***.***> wrote:

> Congratulations! Are you interested in submitting your score for the final
> problem in lab 4 to the unofficial leaderboard?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/qiskit-community/ibm-quantum-challenge-fall-22/pull/31#issuecomment-1328430445>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVJTWUMOCK3ZBV4QHMW2VRTWKQHGHANCNFSM6AAAAAASEQMHKY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> .com>
>

mrvee-qC-bee commented:
Hi! Can you please submit the final lab solution in the solutions-by-participants folder? 😁 You can skip the other lab solutions as they have already been posted in the solutions-by-authors folder. Thank you so much! :D

mvsg2 committed:
Merge branch 'qiskit-community:main' into main

mvsg2 committed:
Add files via upload

NOTE: I actually reran all the cells again (after the Challenge ended). So, the outputs of the cells do not show "Congratulations!...".
On the same note, I also do not remember the exact score.

mvsg2 commented:
Hey! Yes, I would like to submit my Lab solutions. But, I am not able to
make a pull request from my forked copy of the main repository for some
reason.

On Mon, Dec 5, 2022 at 10:35 AM Vishal Bajpe ***@***.***>
wrote:

> Hi! Can you please submit the final lab solution in the
> solutions-by-participants folder? 😁 You can skip the other lab solutions
> as they have already been posted in the solutions-by-authors folder. Thank
> you so much! :D
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/qiskit-community/ibm-quantum-challenge-fall-22/pull/31#issuecomment-1336757259>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVJTWUOLCMLQAPMSM7O7T23WLVZYHANCNFSM6AAAAAASEQMHKY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> .com>
>

mvsg2 committed:
Merge branch 'qiskit-community:main' into main
